backend/controllers/user.controller.ts

```diff
@@ -1,12 +1,11 @@
 import { Request, Response } from 'express';
-import HttpError from '../utils/error';
 import User, { IUser } from '../models/user.model';
 
 export const getAll = async (_req: Request, res: Response) => {
   try {
     const users = await User.find();
     if (users.length === 0) {
-      throw new HttpError('Not found any users', 404);
+      return res.status(403).json({ message: 'Not found any users' });
```

Comment:
Instead of hardcoded status code, please use defined ones in `/utils/StatusCode.ts`. It is more readable when you type `StatusCode.NOT_FOUND` than `404`. Proceed with this for each cases when you used numbers as a status